utils/dockerutil.py

```diff
@@ -284,7 +292,7 @@ def find_cgroup_from_proc(cls, mountpoints, pid, subsys, docker_root='/'):
 
                 # CentOS7 will report `cpu,cpuacct` and then have the path on
                 # `cpuacct,cpu`
-                if 'cpuacct' in mountpoint and 'cpuacct' in subsys:
+                if 'cpuacct' in mountpoint and any(k in subsys for k in ['cpuacct', 'cpu']):
```

Comment:
also as you have just two options it would be more readable to just do
```python
if 'cpuacct' in mountpoint and ('cpuacct' in subsys or 'cpu' in subsys):
```